src/components/Icon/icons/IconGlobe.tsx

```diff
@@ -15,7 +15,7 @@ const IconGlobe = (props: IconProps): JSX.Element => {
     >
       <defs>
         <linearGradient
-          id="New_Gradient_Swatch_3"
+          id="New_Gradient_Swatch_3-3"
```

Comment:
we'll need to update this on L35 too
can we swap to something more meaningful, like iconGlobeAccent?